packages/storage/src/components/UpdateUserPhotoModal/styled.js

```diff
@@ -0,0 +1,158 @@
+import styled from 'styled-components';
+import { Modal, Button } from 'react-rainbow-components';
+
+export const Container = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 2rem;
+    border: 0;
+`;
+
+export const StyledImg = styled.img`
+    border-radius: 50%;
+    width: 200px;
+    height: 200px;
+`;
+
+export const StyledHeader = styled.h1`
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 26px;
+    margin-bottom: 11px;
+    margin-top: -25px;
+
+    @media (min-width: 600px) {
+        padding-right: 90px;
+    }
+`;
+
+export const StyledParagraph = styled.p`
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 150%;
+    margin-bottom: 35px;
+    color: #4f5657e5;
```

Comment:
let's no use colors inline

Reply:
@adrianestevez1 this color here is not valid and also we can't use colors inline